enterprise/setup/index.md

```diff
@@ -0,0 +1,323 @@
+
+Once the system has collected sufficient data it should look similar to this:
+
+![](dashboard_with_data.png)
+
+
+### Step 7: Adding Additional Team Members
+
+For regular setups, you can invite additional team members on a per-database basis, through the Settings screen.
+
+With LDAP authentication you can simply direct your colleagues to the root of your pganalyze URL, an account will be created when they first login.
+
+Note that only administrators can add new databases (see Appendix B).
+
+
+## Appendix
+
+### Appendix A - Amazon Web Services IAM Role
+
+Your IAM role will need two policies assigned - at first, the predefined "AmazonRDSReadOnlyAccess" policy that gives access to most Amazon RDS information and statistics.
+
+In addition, you'll also need to add an inline policy with the following content, if you want to use the log monitoring feature:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "rds:Describe*",
+                "rds:ListTagsForResource",
+                "ec2:DescribeAccountAttributes",
+                "ec2:DescribeAvailabilityZones",
+                "ec2:DescribeSecurityGroups",
+                "ec2:DescribeVpcs"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+        },
+        {
+            "Action": [
+                "cloudwatch:GetMetricStatistics",
+                "logs:DescribeLogStreams",
+                "logs:GetLogEvents"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+        },
+        {
+            "Action": [ "rds:DownloadDBLogFilePortion" ],
+            "Effect": "Allow",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+This is required so that pganalyze can access and download the log files for each instance.
+
+
+### Appendix B - User Roles
+
+Administrator roles:
+* Can create new databases
+* Can access the /admin pages
+* Can modify database settings
+* Can delete databases
+
+Normal user roles:
+* Can view statistics and query information
+* Can view automated Check-Up information, and mark check-ups as acknowledged
+* Receive weekly email reports (if enabled)
+
+Users without valid accounts cannot access any information.
+
+
+### Appendix C - Environment Variables
+
+The following environment variables can be passed into the Docker image. Note that the examples are listed with multiple lines here for formatting reasons, but you need to pass them as one line.
+
+* `DATABASE_URL`: PostgreSQL server used for storing statistics information (required)
+* `LICENSE_KEY`: License key provided to you by the pganalyze team (required)
+* `MAILER_URL`: SMTP server used to send system emails, e.g. user invites (optional, recommended)
+* `AWS_ACCESS_KEY_ID:` AWS Access Key ID (optional)
+* `AWS_SECRET_ACCESS_KEY`: AWS Access Key ID (optional)
```

Comment:
This should be "AWS Secret Access Key (optional)", right?

Reply:
Yup, good catch!